components/form/SizeSelector.tsx

```diff
@@ -1,13 +1,17 @@
 import React, { useEffect, useState } from 'react';
 import { UseFormRegister } from 'react-hook-form';
-import { PartialItem } from '@/types/supabaseTypes';
 
 interface SizeSelectorProps {
   category: string;
-  register: UseFormRegister<PartialItem>;
+  register: UseFormRegister<any>;
```

Comment:
To keep or types strong we don't want to use <any> as type. So please either revert, if you're having trouble getting the types to work. Let us know